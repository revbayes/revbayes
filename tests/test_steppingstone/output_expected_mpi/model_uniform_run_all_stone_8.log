Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-25250.52	-25231.28	-19.23402	1.261321	0.0316531	0.150265	0.0661403	0.160559	0.106123	0.218669	0.713498	0.0962923	0.106784	0.338224	0.204543	0.295012	0.0804474	0.300267	0.27736	0.143953	0.34445	0.0620849	0.205749	0.174448	0.117423	0.241634	0.241479	0.138381	0.00841422	0.00691979	0.198135	0.165487	0.00872913	0.261555	0.087148	0.0733638	0.103411	0.001255	0.0734089	0.0471113	0.0528831	0.141261	0.00935167	0.0260204	0.152891	0.207095	0.166299	0.0498381	0.4124	0.0464441	0.0306795	0.381325	0.0793131	0.18599	0.543321	1.03268	2.23801	0.148624	0.337973	0.117557	0.395846	0.6651382	6.606178
10	-28583.41	-28579.3	-4.117228	0.7442987	0.111422	0.101845	0.0159858	0.0499178	0.0520319	0.166705	0.0713973	0.0869892	0.180822	0.0685926	0.126334	0.193449	0.0572351	0.121953	0.22245	0.133729	0.0859271	0.0408788	0.18132	0.295809	0.0356966	0.30191	0.172178	0.186634	0.0418096	0.0130132	0.131654	0.0762849	0.00453983	0.137754	0.0530497	0.0640638	0.0853147	0.00363994	0.207007	0.227398	0.0149089	0.232085	0.00444205	0.0232142	0.135421	0.241238	0.278406	0.355806	0.332668	0.0864	0.0289621	0.108716	0.0874486	0.0836646	0.383703	0.94024	2.59239	0.0224145	0.402479	0.123725	0.451382	0.2259556	5.036457
