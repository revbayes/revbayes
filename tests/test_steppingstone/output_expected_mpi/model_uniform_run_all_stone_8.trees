Iteration	Posterior	Likelihood	Prior	psi
0	-25250.52	-25231.28	-19.23402	((Aotus_trivirgatus[&index=1]:0.031653,((Saimiri_sciureus[&index=20]:0.174448,(Hylobates_lar[&index=9]:0.106784,Pan_paniscus[&index=17]:0.344450)[&index=42]:0.207095)[&index=43]:0.166299,((Macaca_mulatta[&index=13]:0.080447,Callicebus_donacophilus[&index=2]:0.150265)[&index=25]:0.008414,(Colobus_guereza[&index=6]:0.218669,Chlorocebus_aethiops[&index=5]:0.106123)[&index=32]:0.073364)[&index=30]:0.261555)[&index=31]:0.087148)[&index=34]:0.001255,Galeopterus_variegatus[&index=23]:0.241479,(Daubentonia_madagascariensis[&index=7]:0.713498,((Tarsius_syrichta[&index=21]:0.117423,(((Propithecus_coquereli[&index=19]:0.205749,((Microcebus_murinus[&index=14]:0.300267,((Lepilemur_hubbardorum[&index=11]:0.204543,(Perodicticus_potto[&index=18]:0.062085,Galago_senegalensis[&index=8]:0.096292)[&index=41]:0.152891)[&index=39]:0.009352,Lemur_catta[&index=10]:0.338224)[&index=38]:0.141261)[&index=40]:0.026020,Varecia_variegata_variegata[&index=22]:0.241634)[&index=28]:0.165487)[&index=27]:0.198135,Loris_tardigradus[&index=12]:0.295012)[&index=29]:0.008729,(Cheirogaleus_major[&index=4]:0.160559,Nycticebus_coucang[&index=15]:0.277360)[&index=37]:0.052883)[&index=35]:0.073409)[&index=24]:0.138381,(Cebus_albifrons[&index=3]:0.066140,Otolemur_crassicaudatus[&index=16]:0.143953)[&index=26]:0.006920)[&index=36]:0.047111)[&index=33]:0.103411)[&index=44];
10	-28583.41	-28579.3	-4.117228	(Galeopterus_variegatus[&index=23]:0.172178,(((Lemur_catta[&index=10]:0.068593,Otolemur_crassicaudatus[&index=16]:0.133729)[&index=38]:0.232085,(Cebus_albifrons[&index=3]:0.015986,(Macaca_mulatta[&index=13]:0.057235,(Callicebus_donacophilus[&index=2]:0.101845,Aotus_trivirgatus[&index=1]:0.111422)[&index=30]:0.137754)[&index=26]:0.013013)[&index=34]:0.003640)[&index=25]:0.041810,((((Cheirogaleus_major[&index=4]:0.049918,Varecia_variegata_variegata[&index=22]:0.301910)[&index=36]:0.227398,Pan_paniscus[&index=17]:0.085927)[&index=37]:0.014909,(Colobus_guereza[&index=6]:0.166705,Chlorocebus_aethiops[&index=5]:0.052032)[&index=32]:0.064064)[&index=24]:0.186634,((Nycticebus_coucang[&index=15]:0.222450,(Propithecus_coquereli[&index=19]:0.181320,(Tarsius_syrichta[&index=21]:0.035697,(Lepilemur_hubbardorum[&index=11]:0.126334,((Perodicticus_potto[&index=18]:0.040879,Galago_senegalensis[&index=8]:0.086989)[&index=40]:0.023214,(Microcebus_murinus[&index=14]:0.121953,(Loris_tardigradus[&index=12]:0.193449,Hylobates_lar[&index=9]:0.180822)[&index=31]:0.053050)[&index=42]:0.241238)[&index=39]:0.004442)[&index=41]:0.135421)[&index=28]:0.076285)[&index=27]:0.131654)[&index=33]:0.085315,Daubentonia_madagascariensis[&index=7]:0.071397)[&index=29]:0.004540)[&index=43]:0.278406)[&index=35]:0.207007,Saimiri_sciureus[&index=20]:0.295809)[&index=44];
