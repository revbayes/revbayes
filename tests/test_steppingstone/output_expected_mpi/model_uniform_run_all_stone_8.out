state	power	likelihood
4	0.00032	-28208.8
5	0.00032	-28267.8
6	0.00032	-28561.4
7	0.00032	-28165.8
8	0.00032	-28479
9	0.00032	-29252.6
10	0.00032	-28579.3
